public/app/features/plugins/extensions/getPluginExtensions.ts

```diff
@@ -58,6 +69,30 @@ export const getPluginExtensions: GetExtensions = ({ context, extensionPointId,
 
         extensions.push(extension);
       }
+
+      // ELEMENT
+      if (isPluginExtensionComponentConfig(extensionConfig)) {
+        // Run the configure() function with the current context, and apply the ovverides
+        const overrides = getComponentExtensionOverrides(registryItem.pluginId, extensionConfig, frozenContext);
+
+        // configure() returned an `undefined` -> hide the extension
+        if (extensionConfig.configure && overrides === undefined) {
+          continue;
+        }
```

Comment:
I think this (and the `getComponentExtensionsOverrides` function) might be left over from the previous approach that used a configure function? 🤔

Reply:
Good catch! 👍

Reply:
Fixed in https://github.com/grafana/grafana/pull/68600/commits/2fa000efc958541cfb01ad7fe84f937f6d933b67.